CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Upgrade mongoose dependence version to 4.13.3
+- Fix bug in mergeArrays function of deviceService (#571)
```

Comment:
Looking to the changes in the code it seems the fix is about internalAttributes, so, maybe the following would be more precise?

> Fix bug in mergeArrays function of deviceService, precluding updates to work when internalAttributes of type Array were used (#571)

Reply:
Fixed in 9fa97a4d08a4b2abc620958d5455a2afcd313531